### PR TITLE
Updated missing params in docstrings

### PR DESCRIPTION
**Pull request by Steve-Cheney**
Howdy y'all, working on a tool to convert packages to JSON and noticed there were some functions missing their parameters in the docstring. I've updated a few that were missing for better pydoc accessibility. All param descriptions were either copied from other similar param or made based on their functionality.

- [x] Tests included or not required because:

Docstring updates only

- [x] Release notes not necessary because:

No changes to logic

### CODE REVIEW

**Comment by Steve-Cheney** [comment]
pre-commit.ci autofix

**Comment by flying-sheep** [comment]
Nice!

One thing: the `ScanpyConfig` class isn‘t supposed to be instantiated by users, so it shouldn’t have a `Parameters` block.

So please revert this part. Otherwise this looks great, thank you.

File: scanpy/_settings.py [hunk 1]
```diff
@@ -91,6 +91,41 @@ def _type_check(var: Any, varname: str, types: type | tuple[type, ...]):
...
+    max_memory
+        Maximum memory usage in Gigabyte.
+    n_jobs
+        Default number of jobs/ CPUs to use for parallel computing.
+    plot_suffix
+        Global suffix that is appended to figure filenames.
+    verbosity
+        Verbosity level (default warning)
+    writedir
+        Directory where the function scanpy.write writes to by default.
```
Comment (flying-sheep) [inline scanpy/_settings.py]:
```suggestion
```

**Review by flying-sheep — CHANGES_REQUESTED** [review]

**Comment by Steve-Cheney** [comment]
Done @flying-sheep !

**Review by flying-sheep — APPROVED** [review]

**Pull request merged**